client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -66,6 +66,11 @@ const SubscriberListContainer = ( {
 							className={ `subscriber-list-container__subscriber-count ${
 								isLoading ? 'loading-placeholder' : ''
 							}` }
+							title={
+								total > 1000
+									? formatNumber( total, getLocaleSlug() || undefined, { notation: 'standard' } )
+									: ''
+							}
```

Comment:
Nit: Doing it this way for `total <= 1000` this `span` element still gets the `title` attribute. It's empty but it's still there. I would suggest sth like this instead:
```suggestion
							title={
								total > 1000
									? formatNumber( total, getLocaleSlug() || undefined, { notation: 'standard' } )
									: undefined
							}
```